doc/releases.md

```diff
@@ -11,11 +11,11 @@ Release authorization is given by the node.js TSC. Once authorized, an individua
 
 There are three relevant Jenkins jobs that should be used for a release flow:
 
-**a.** **Test runs:** **[iojs+any-pr+multi](https://jenkins-iojs.nodesource.com/job/iojs+any-pr+multi/)** is used for a final full-test run to ensure that the current *HEAD* is stable.
+**a.** **Test runs:** **[iojs+any-pr+multi](https://ci.nodejs.org/job/iojs+any-pr+multi/)** is used for a final full-test run to ensure that the current *HEAD* is stable.
```

Comment:
Fixed & rebased over `master`, here and in one other place below.

I assume `iojs+release` is still used?